src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Truck truck = new Truck();
+        Excavator excavator = new Excavator();
+        Bulldozer bulldozer = new Bulldozer();
```

Comment:
According to the checklist, it is recommended to use abstract references instead of specific ones to take advantage of polymorphism. Instead of creating `Truck`, `Excavator`, and `Bulldozer` as specific types, you should declare them as `Machine` type. This allows for more flexible code if you decide to add more functionality that works with different `Machine` types.